### DISCUSSION TITLE
How to setup devtools while using solidjs with Astro ?

### DISCUSSION THREAD

**Discussion by geminigeek** [solidjs/solid#1488] [Q&A]
hi,

can anyone help me setup chrome devtools with astro integration ?

**Comment by barel-mishal** [comment]
I am willing to trying help you. please get back to me on linkedin https://www.linkedin.com/in/barel-mishal/

**Reply by geminigeek** [reply]
i am not on linkedin !!

**Reply by barel-mishal** [reply]
well, I think you can try also give a some other why to contact with you or you do a LinkedIn

**Reply by samuelstroschein** [reply]
@barel-mishal why don't you help @geminigeek in this discussion?

**Reply by barel-mishal** [reply]
Hi that possible. Why not actually @samuelstroschein  :)
I just thought it will be fester do it on a platform that the communication is more direct. 

@geminigeek
BTW You can start providing more information what you have tried and where the challenge for you.

And also I want to remind that I was start by saying that I can try and also now I want to clarify that I do not really know how to do that and if that possible or not but from my experience I setup another projects I can help. 
@samuelstroschein If you know please give your help too. 

Sorry if my offered is not appropriate, from you asking, I can understand that.
I am just started to try help on the open source, so I am still learning how done.

**Reply by ryansolid** [reply]
Benefit of using discissions is others can see. So answering here might help anyone having similar issues.
